deploy/helm/sumologic/conf/metrics/otelcol/processors.yaml

```diff
@@ -126,10 +126,10 @@ resource/delete_source_metadata:
       key: _sourceHost
     - action: delete
       key: _sourceName
-resource/remove_k8s_pod_pod_name:
+resource/remove_pod_name:
   attributes:
     - action: delete
-      key: k8s.pod.pod_name
+      key: pod
```

Comment:
Two things:
1. I don't think this worked before, the actual name set by k8sprocessor is `k8s.pod.name`.
2. After moving this, the name to be deleted changes, because sumologic_schema translates it.

Reply:
Integration tests are failing because of this change. Does anyone know what this processor was supposed to do?

Reply:
It seems to be some antique change: https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/1786

Reply:
I think `k8s.pod.pod_name` is added by source processor. Is there a problem moving this processor to right after source processor in the pipeline?

Reply:
Ok, it makes sense now. 
I'd prefer not moving this directly after source processor to make changes in this PR as little intrusive as possible (if I moved this, then, for example, if someone used key `k8s.pod.pod_name` in extra processors, it would break the config). But we can move it.